extras/drift_postgres/lib/src/pg_database.dart

```diff
@@ -93,13 +93,19 @@ class _PgDelegate extends DatabaseDelegate {
       result = await _ec.query(statement);
     } else {
       result = await _ec.query(statement,
-          substitutionValues: args
-              .asMap()
-              .map((key, value) => MapEntry((key + 1).toString(), value)));
+          substitutionValues: args.asMap().map((key, value) =>
```

Comment:
This looks good to me, but don't `runBatched` and `_runWithArgs` need the same transformation?

Reply:
You sure, I missed.